pandas/tests/io/formats/test_to_csv.py

```diff
@@ -223,3 +223,27 @@ def test_to_csv_multi_index(self):
 
         exp = "foo\nbar\n1\n"
         assert df.to_csv(index=False) == exp
+
+    def test_to_csv_string_array(self):
+        # GH 10813
+        str_array = [{'names': ['foo', 'bar']}, {'names': ['baz', 'qux']}]
+        df = pd.DataFrame(str_array)
+        expected_ascii = '''\
+,names
```

Comment:
so if you make this 2 test functions, then you can xfail the non-working one (to at least get things passing)